msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/update/TestCaseUpdateCompanyPortal.kt

```diff
@@ -47,14 +47,14 @@ import java.util.*
 @LTWTests
 class TestCaseUpdateCompanyPortal : AbstractMsalCustomBrokerInstallationTest() {
 
-    private val mCompanyPortal: BrokerCompanyPortal = installOldCompanyPortal()
```

Comment:
was it not installing the old CP pk correctly when this step executes?

Reply:
It was installing it before the test rules would run, so if these tests were ran before any others, the copy file rule would not have been ran at any point, and they would fail because the files are not where they need to be.